base/ltoutput.dtx

```diff
@@ -4568,7 +4835,18 @@
 % Sets |@tempswa| true if there is room for |\@currbox|.
 %
 %    \begin{macrocode}
+%</2ekernel|fltrace>
+%<*2ekernel|fltrace|latexrelease>
+%<latexrelease|fltrace>\IncludeInRelease{2025/06/01}%
+%<latexrelease|fltrace>                 {\@flcheckspace}{float placement calculation}%
 \def \@flcheckspace #1#2{%
+%    \end{macrocode}
+%    When this test is executed the value of \cs{@reqcolroom} may be
+%    not longer correct due to a previous test that altered it. We
```

Comment:
. . . executed, the . . . 

And:
may be not longer be correct 
  ==>
    may no longer be correct